bin/fasta-to-phylip.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+
+import sys
+import tempfile
+
+from dark.fasta import FastaReads
+from dark.fasta_ss import SSFastaReads
+from dark.fastq import FastqReads
+
+
+if __name__ == '__main__':
+    import argparse
+
+    parser = argparse.ArgumentParser(
+        description=('Given FASTA on stdin write the sequence ids and lengths '
+                     'to stdout.'))
+
+    parser.add_argument(
+        '--readClass', default='fasta', choices=('fasta', 'fastq', 'fasta-ss'),
```

Comment:
I find the name confusing. I think we use `readClass` for specifying `DNARead` or `AARead` in `FilterFasta`. Can we use a different name, like `sequenceFileType` or something?